src/Helpers/VulnerabilitiesHelper.test.js

```diff
@@ -86,4 +156,122 @@ describe('VulnerabilitiesHelper', () => {
+        const cveList = {
+            isLoading: false,
+            payload: {
+                meta: {
+                    test: 'test'
+                },
+                data: [
+                    {
+                        type: 'cve',
+                        id: 'CVE-2019-6116',
+                        attributes: {
+                            business_risk: "Not Defined",
+                            business_risk_id: 0,
+                            business_risk_text: null,
+                            cve_status_id: 0,
+                            cve_status_text: null,
+                            cvss2_score: null,
+                            cvss3_score: "7.500",
+                            description: "A network amplification",
+                            impact: "Important",
+                            public_date: "2020-05-19T00:00:00+00:00",
+                            reporter: 1,
+                            rule: null,
+                            status: "Scheduled for Patch",
+                            status_id: 3,
+                            status_text: "pair status different",
+                            synopsis: "CVE-2020-12662"
+                        }
+                    }
+                ]
+            }
+        };
+        expect(JSON.stringify(createCveListBySystem(null, cveList))).toMatchSnapshot();
+    });
+
+    it('test function createCveListBySystem() render CSaw icon', () => {
+        const cveList = {
+            isLoading: false,
+            payload: {
+                meta: {
+                    test: 'test'
+                },
+                data: [
+                    {
+                        type: 'cve',
+                        id: 'CVE-2019-6454',
+                        attributes: {
+                            business_risk: "Not Defined",
+                            business_risk_id: 0,
+                            business_risk_text: null,
+                            cve_status_id: 2,
+                            cve_status_text: "testhello",
+                            cvss2_score: null,
+                            cvss3_score: "6.500",
+                            description: "A new domain bypass",
+                            impact: "Moderate",
+                            public_date: "2020-06-09T17:00:00+00:00",
+                            reporter: 1,
+                            rule: null,
+                            status: "On-Hold",
+                            status_id: 2,
+                            status_text: "testhello",
+                            synopsis: "CVE-2020-0543",
+                            rule: 'testRule'
+                        }
+                    }
+                ]
+            }
+        };
+
+        const result = createCveListBySystem(null, cveList);
+        expect(result.data[0].cells[0].title.props.children[0]).toEqual(expect.anything());
```

Comment:
`expect.anything()` will match everything except `null` and `underfined` is this what you wanted?